flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/ExecutionCheckpointingOptions.java

```diff
@@ -88,16 +88,18 @@
                                     + "we do not tolerance any checkpoint failure.");
 
     public static final ConfigOption<CheckpointConfig.ExternalizedCheckpointCleanup>
-            EXTERNALIZED_CHECKPOINT =
+            EXTERNALIZED_CHECKPOINT_CLEANUP =
```

Comment:
This is a breaking change. If it isn't necessary, we should avoid it; otherwise we can keep a deprecated alias for now.